### PR TITLE
BREAKING CHANGE: 0.29

### PR DESCRIPTION
**Pull request by ovr**
Hello!

Thanks

### CODE REVIEW

File: DEPRECATION.md [hunk 1]
```diff
@@ -56,8 +58,6 @@ features:
 Node.js 8 reached [End of Life on December 31, 2019][link-nodejs-eol]. This
 means no more updates. Please upgrade to Node.js 10 or higher.
 
-[link-nodejs-eol]: https://github.com/nodejs/Release#end-of-life-releases
-
```
Comment (hassankhan) [inline DEPRECATION.md:60]: This breaks the link for Node 10, we need to keep one of them

File: DEPRECATION.md [hunk 2]
```diff
@@ -44,10 +44,12 @@ features:
 | Deprecated | [Absolute import for `@cubejs-backend/server-core`](#absolute-import-for-@cubejs-backendserver-core)               | v0.25.4    | v0.30.0   |
 | Deprecated | [Absolute import for `@cubejs-backend/schema-compiler`](#absolute-import-for-@cubejs-backendschema-compiler)       | v0.25.21   | v0.32.0   |
 | Deprecated | [`checkAuthMiddleware`](#checkauthmiddleware)                                                                      | v0.26.0    |           |
-| Deprecated | [Node.js 10](#nodejs-10)                                                                                           | v0.26.0    |           |
+| Deprecated | [Node.js 10](#nodejs-10)                                                                                           | v0.26.0    | v0.29.0   |
```
Comment (hassankhan) [inline DEPRECATION.md:47]: Shouldn't this be `Removed` instead of `Deprecated`?

File: DEPRECATION.md [hunk 3]
```diff
@@ -44,10 +44,12 @@ features:
...
+| Deprecated | [Node.js 15](#nodejs-15)                                                                                           | v0.26.0    | v0.29.0   |
```
Comment (hassankhan) [inline DEPRECATION.md:48]: Shouldn't this be `Removed` instead of `Deprecated`?

**Review by hassankhan — CHANGES_REQUESTED** [review]
Minor details in the `DEPRECATION.md` file

File: DEPRECATION.md (hunk 3 above)
Comment (ovr) [reply DEPRECATION.md]: Yes, let's use Removed. Fixed, Thanks!

**Pull request merged**